packages/core/src/lib/garden/garden.spec.ts

```
@@ -66,54 +66,72 @@ describe('swap and execute using garden', () => {
+  it.skip('initialize garden from wallets', async () => {
+    const garden = Garden.fromWallets({
+      environment: Environment.TESTNET,
+      digestKey:
+        '7fb6d160fccb337904f2c630649950cc974a24a2931c3fdd652d3cd43810a857',
+      wallets: {
+        evm: arbitrumWalletClient,
+      },
+    });
 
-  // let wallets: Partial<{ [key in Chain]: WalletClient }> = {};
+    const garden = new Garden({
```

Biome (1.9.4)

[error] Shouldn't redeclare 'garden'. Consider to delete it or rename it.

'garden' is defined here:


(lint/suspicious/noRedeclare)